pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
Great, here's my branch: https://github.com/python/pythondotorg/compare/master...berkerpeksag:pr/894 If it looks good to both of you, I can merge and deploy it.